src/BookReader.js

```diff
@@ -662,93 +673,146 @@ BookReader.prototype.resize = function() {
 };
 
 /**
- * Binds keyboard event listeners
+ * Intersection observer callback sets BookReader keyboard focus
+ * off when the BookReader is not in the viewport.
+ * @param {object} entries
+ * @param {function} observer
  */
-BookReader.prototype.setupKeyListeners = function() {
-  var self = this;
+BookReader.prototype.handleIntersection = function (entries, observer) {
```

Comment:
This doesn't seem significant enough to get it's own function? Inlining it would be clearer for me. It also doesn't make too much sense out of context, since I don't know what element this intersection observer is currently being applied to.